code/modules/mob/living/carbon/human/species/shadow.dm

```diff
@@ -16,7 +17,7 @@
 		"brain" = /obj/item/organ/brain
 		)
 
-	flags = NO_BLOOD | NO_BREATHE | NO_SCAN
+	flags = NO_BLOOD | NO_BREATHE | NO_SCAN | NO_POISON
```

Comment:
I don't really have much of an opinion on this one, people rarely play as shadow people because they die horribly in the light, and (although this might've been fixed by now) last time I turned into one, they did not have proper night vision.